js/weatherWidget.js

```diff
@@ -0,0 +1,38 @@
+// Fetches weather data and updates - External API documentation: https://rapidapi.com/community/api/open-weather13
+
+
+async function fetchWeather() {
+    const url = 'https://open-weather13.p.rapidapi.com/city/sydney';
```

Comment:
You could think about a way to filter through different cities. There are also webAPIs that could determine the user's location.